app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -29,12 +33,28 @@
     // enum values with DEFAULT actions //
     //////////////////////////////////////
 
-    show_channel_details(R.string.show_channel_details, (fragment, item) ->
-        // For some reason `getParentFragmentManager()` doesn't work, but this does.
-        NavigationHelper.openChannelFragment(
-                fragment.requireActivity().getSupportFragmentManager(),
-                item.getServiceId(), item.getUploaderUrl(), item.getUploaderName())
-    ),
+    show_channel_details(R.string.show_channel_details, (fragment, item) -> {
+        if (isNullOrEmpty(item.getUploaderUrl())) {
+            final int serviceId = item.getServiceId();
+            final String url = item.getUrl();
+            // TODO: Some visual loading indicator
```

Comment:
I don't know what to suggest; maybe it can do without a loading indicator (similar to the "Mark as watched" action present in 0.21.9).

Reply:
I think there should be _some_ kind of indicator (even just a `Toast`)